components/bl-granim-component/component.json

```diff
@@ -0,0 +1,358 @@
+          "label": "none"
+        },
+        {
+          "value": "stretch",
+          "label": "stretch"
+        },
+        {
+          "value": "stretch-if-smaller",
+          "label": "stretch-if-smaller"
+        },
+        {
+          "value": "stretch-if-bigger",
+          "label": "stretch-if-bigger"
+        }
+      ],
+      "settingTooltip": "Stretching the image horizontally"
+    },
+    {
+      "type": "select",
+      "name": "stretchModeY",
+      "label": "Stretch Mode Y",
+      "showInSettings": true,
+      "defaultValue": "none",
+      "hasLogicHandler": false,
+      "handlerId": "stretchModeYLogic",
+      "options": [
+        {
+          "value": "none",
+          "label": "none"
+        },
+        {
+          "value": "stretch",
+          "label": "stretch"
+        },
+        {
+          "value": "stretch-if-smaller",
+          "label": "stretch-if-smaller"
+        },
+        {
+          "value": "stretch-if-bigger",
+          "label": "stretch-if-bigger"
+        }
+      ],
+      "settingTooltip": "Stretching the image vertically"
+    },
+    {
+      "type": "select",
+      "name": "blendingMode",
+      "label": "Blending Mode",
+      "showInSettings": true,
+      "defaultValue": "multiply",
+      "hasLogicHandler": false,
+      "handlerId": "blendingModeLogic",
+      "options": [
+        {
+          "value": "multiply",
+          "label": "multiply"
+        },
+        {
+          "value": "screen",
+          "label": "screen"
+        },
+        {
+          "value": "normal",
+          "label": "normal"
+        },
+        {
+          "value": "overlay",
+          "label": "overlay"
+        },
+        {
+          "value": "darken",
+          "label": "darken"
+        },
+        {
+          "value": "lighten",
+          "label": "lighten"
+        },
+        {
+          "value": "lighter",
+          "label": "lighter"
+        },
+        {
+          "value": "color-dodge",
+          "label": "color-dodge"
+        },
+        {
+          "value": "color-burn",
+          "label": "color-burn"
+        },
+        {
+          "value": "hard-light",
+          "label": "hard-light"
+        },
+        {
+          "value": "soft-light",
+          "label": "soft-light"
+        },
+        {
+          "value": "difference",
+          "label": "difference"
+        },
+        {
+          "value": "exclusion",
+          "label": "exclusion"
+        },
+        {
+          "value": "hue",
+          "label": "hue"
+        },
+        {
+          "value": "saturation",
+          "label": "saturation"
+        },
+        {
+          "value": "color",
+          "label": "color"
+        },
+        {
+          "value": "luminosity",
+          "label": "luminosity"
+        }
+      ],
+      "settingTooltip": "Options for blending images with a gradient"
+    },
+    {
+      "type": "checkbox",
+      "name": "isPausedWhenNotInView",
```

Comment:
How about "Optimized" or "Offscreen Pause"?

Reply:
Yes, the parameter was renamed to "Offscreen Pause" to be displayed to the user in the settings, but there is no point in changing the internal name, in this form it is transferred inside the library.